decimal.go

```diff
@@ -52,6 +52,14 @@ var DivisionPrecision = 16
 // silently lose precision.
 var MarshalJSONWithoutQuotes = false
 
+// StringTrimTrailingZeros should be set to false if you want the decimal stringify without zeros trailing.
+// By default, when decimal is output as a string (for example, in JSON), zeros are truncated from it (2.00 -> 2, 3.11 -> 3.11, 13.000 -> 13).
+// But this logic can be changed by this variable.
+// For example, if you have numeric(10,2) values stored in your database,
+// and you want your API response to always be given 2 decimal places (even 2.00, 3.00, 17.00 [not 2,3,17]),
+// then this variable is a great way out.
```

Comment:
```suggestion
// TrimTrailingZeros specifies whether trailing zeroes should be trimmed from a string representation of decimal.
// If set to true, trailing zeroes will be truncated (2.00 -> 2, 3.11 -> 3.11, 13.000 -> 13),
// otherwise trailing zeroes will be preserved (2.00 -> 2.00, 3.11 -> 3.11, 13.000 -> 13.000).
// Setting this value to false can be useful for APIs where exact decimal string representation matters.
```

Reply:
What do you think about it? A bit simplified and a bit more explicit version of your suggestion.